back-office/design/themes-and-logo/theme-logo.md

```diff
@@ -39,39 +52,41 @@ IMPROVEMENT to do : when i import a logo it’s not resized it must be to avoid
 All logo images should be imported via the upload field, with the placeholder _Choose file(s)_ and the CTA _Browse_, all being clickable items opening the user's local files. Available logo images extensions are: .gif, .jpg, .jpeg, .jpe, .png, .svg.
 
 
-### Visit the theme catalog button
-
-The button “Visit the theme catalog” redirect to this link
+### Multistore behavior
 
+In a multistore context, the options listed above have [checkboxes](https://github.com/PrestaShop/PrestaShop/issues/19375) and [specific settings drop-down](https://github.com/PrestaShop/PrestaShop/issues/19327), take a look at the conditions and specs [here](https://github.com/PrestaShop/prestashop-specs/blob/master/back-office/multistoregeneralspecs.md)
 
-## Theme installed
+If I am in an all shops context or in a group of shops context, an information notification is displayed: `You must select a shop from the above list if you wish to choose a theme.`.
 
-If i import a theme and if i select a shop a section appear on the bottom of the page after hover, a button use this theme appear,on click, the current theme is switched with the one selected and the preview of theme +used change with the theme chosen.
+The multistore yes / no toggle allowing to select all / unselect all checkboxes should be removed.
 
-If i don’t select a shop and i select the group of shop this appear instead of the theme section:
+## My themes 
 
+All the themes imported are displayed here with a preview.
 
-## Multistore state
+A button "use this theme" is displayed for themes not currently used. After clicking on it, the current theme is switched with the one selected and "My current theme" is displayed for the theme used.
 
-If i activate multi store, multi store section will appear only if i click on a shop\(here: derek ortega or testshop\) or i select the group of shop\(here: default group\) \(except all shops\)
+A button "delete" is displayed for themes not currently used. After clicking on it, a modal is displayed: "Delete this theme ?". A button "Cancel" and a "cross" allow canceling the deletion. A button "Yes" allows confirming the deletion. A message "Successful deletion." is displayed to confirm the deletion.
```

Comment:
```suggestion
A button "delete" is displayed for themes not currently used. After clicking on it, a modal is displayed: "Delete this theme?". A button "Cancel" and a "cross" allow canceling the deletion. A button "Yes" allows confirming the deletion. A message "Successful deletion." is displayed to confirm the deletion.
```

Currently there is no "cross" on the confirmation modal, is this a new wanted behavior or should the spec be updated ?

Reply:
You are right, I deleted it and updated the wording btw: https://github.com/PrestaShop/prestashop-specs/pull/195/commits/b85dd8ebde85f44c1baa48aedd2ad9bfdf8b2e15